In [1]:
import re

leg = open('/Users/shivangi/Documents/14216869.txt') # remember to change the filepath to the path on your computer
count = 0
for line in leg:
    line = line.rstrip() #remove the white spaces etc at the end of the line
    if re.search('negative', line):
        if re.search('1', line):
            count = count + 1
            print(re.findall('([a-zA-Z]+).',line)[0])

abandon
abandoned
abandonment
abduction
aberrant
aberration
abhor
abhorrent
abject
abnormal
abolish
abolition
abominable
abomination
abort
abortion
abortive
abrasion
abrogate
abscess
absence
absent
absentee
absenteeism
absurd
absurdity
abundance
abuse
abysmal
abyss
accident
accidental
accursed
accusation
accusative
accused
accuser
accusing
ache
aching
acid
actionable
adder
addiction
adipose
adjudicate
admonition
adrift
adulterated
adultery
adversary
adverse
adversity
afflict
afflicted
affliction
affront
afraid
aftermath
aftertaste
aggravated
aggravating
aggravation
aggression
aggressive
aggressor
aghast
agitated
agitation
agonizing
agony
aground
ail
ailing
aimless
airs
alarm
alarming
alcoholism
alien
alienate
alienated
alienation
alimony
allegation
allege
aloof
altercation
ambiguous
ambush
amnesia
amphetamines
anaconda
anal
anarchism
anarchist
anarchy
anathema
ancient
anger
angina
angling
angry
anguish
animosity
animus
annihilate
annihilated
annihilation
annoy
annoyance
annoying
annul


hunting
hurricane
hurried
hurt
hurtful
hurting
hustler
hydra
hydrocephalus
hype
hyperbole
hypocrisy
hypocrite
hypocritical
hysteria
hysterical
idiocy
idiot
idiotic
idler
idolatry
ignorance
ignorant
ignore
ill
illegal
illegality
illegible
illegitimate
illicit
illiterate
illness
illogical
illusion
imitated
imitation
immature
immaturity
immediately
immoral
immorality
immovable
impair
impairment
impassable
impatience
impatient
impeach
impeachment
impede
imperfection
imperfectly
impersonate
impersonation
implacable
implicate
impolite
imposition
impossible
impotence
impotent
impound
impracticable
imprison
imprisoned
imprisonment
impropriety
imprudent
impure
impurity
imputation
inability
inaccessible
inaccurate
inaction
inactivity
inadequacy
inadequate
inadmissible
inane
inapplicable
inappropriate
inattention
inaudible
incalculable
incapacity
incarceration
incase
incendiary
incense
incessant
incest
incestuous
incineration
incite
inclement
incoherent
income
incompatible
incompetence
incompeten

tolerate
toothache
torment
torn
torpedo
torrid
tort
tortious
torture
touched
touchy
tough
toxic
toxin
tragedy
tragic
traitor
tramp
trance
transgression
trappings
traps
trash
trashy
traumatic
travail
travesty
treacherous
treachery
treason
treat
trembling
tremor
trepidation
trespass
tribulation
tribunal
trick
trickery
trifle
tripping
troll
troublesome
trumpet
tumble
tumor
tumour
tumult
tumultuous
turbulence
turbulent
turmoil
twitch
typhoon
tyrannical
tyranny
tyrant
ugliness
ugly
ulcer
ulterior
ultimatum
unable
unacceptable
unaccountable
unapproved
unattached
unattainable
unattractive
unauthorized
unavoidable
unaware
unbearable
unbeaten
unbelief
unbelievable
unborn
unbridled
uncanny
uncaring
uncertain
unchangeable
unclean
uncomfortable
unconscionable
unconscious
unconstitutional
uncontrollable
uncontrolled
undecided
undermined
underpaid
undersized
undesirable
undesired
undo
uneasiness
uneasy
uneducated
unemployed
unequal
uneven
unexpected
unexplained
unfair
unfairness
unfaithful
unfavorab

In [2]:
count

3325

As seen above, below code should be used to find all negative words in NRC emoticon lexicon.

leg = open('/Users/shivangi/Documents/14216869.txt') # remember to change the filepath to the path on your computer
count = 0
for line in leg:
    line = line.rstrip() #remove the white spaces etc at the end of the line
    if re.search('negative', line):
        count = count + 1
        print(re.findall('([a-zA-Z]+).',line)[0])
# print(count)

Total number of negative words in NRC lexicon = 3325

Total negative words are included in the LIWC 2007 lexicon = 57

Some examples of negative words that are in NRC but not in LIWC =

aback
abacus
abate
abbot
abbreviate